minichain/p2p.py

```
@@ -101,70 +129,185 @@ def peer_count(self) -> int:
+                        if not line:
+                            continue
                         try:
-                            msg = json.loads(line.decode().strip())
-                            msg["_peer_addr"] = addr
-                            self._to_asyncio.put(("MSG", msg))
-                        except Exception: pass
-            except Exception: pass
+                            parsed = json.loads(line.decode().strip())
+                            parsed["_peer_addr"] = addr
+                            self._to_asyncio.put(("MSG", parsed))
+                        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)